Gemfile

```
@@ -53,6 +53,8 @@ gem 'tzinfo', '~> 1.2'
 gem 'open_id_authentication'
 gem 'protected_attributes_continued', '~> 1.8.2'
 gem 'ruby-openid'
+gem 'fog-google'
```

We could do this due to https://github.com/fog/fog-google/issues/535

```suggestion
gem 'fog-google', '1.13.0' # Limited to `v1.13.0` due to https://github.com/fog/fog-google/issues/535
```